src/docfx/lib/HtmlUtility.cs

```diff
@@ -144,10 +145,11 @@ public static string TransformLinks(string html, Func<string, int, string> trans
             return result.ToString();
         }
 
-        public static string TransformXref(string html, Func<string, bool, int, (string? href, string display)> transform)
+        public static string TransformXref(string html, MarkdownObject block, Func<SourceInfo<string>?, SourceInfo<string>?, bool, (string? href, string display)> resolveXref)
         {
             // Fast pass it does not have <xref> tag
-            if (!(html.Contains("<xref", StringComparison.OrdinalIgnoreCase) && html.Contains("href", StringComparison.OrdinalIgnoreCase)))
+            if (!(html.Contains("<xref", StringComparison.OrdinalIgnoreCase)
+                && (html.Contains("href", StringComparison.OrdinalIgnoreCase) || html.Contains("uid", StringComparison.OrdinalIgnoreCase))))
```

Comment:
` href`/` uid` (with a leading space) would be better?

Reply:
html attribute can also be `attr = 'value'`, with space between `attr`, `=` and `value`
```
<!DOCTYPE html>↩
<html>↩
<head>↩
<title>xxx</title>↩
</head>↩
<body>↩
↩
<img src = "src" alt = "xxxx"/>↩
↩
↩
</body>↩
</html>
```